[2021-03-22 17:17:29,311] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: test_slack.send_slack 2021-03-22T07:08:00+00:00 [queued]>
[2021-03-22 17:17:29,321] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: test_slack.send_slack 2021-03-22T07:08:00+00:00 [queued]>
[2021-03-22 17:17:29,321] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 17:17:29,321] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-22 17:17:29,321] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 17:17:29,327] {taskinstance.py:1063} INFO - Executing <Task(SlackAPIPostOperator): send_slack> on 2021-03-22T07:08:00+00:00
[2021-03-22 17:17:29,330] {standard_task_runner.py:52} INFO - Started process 16687 to run task
[2021-03-22 17:17:29,338] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test_slack', 'send_slack', '2021-03-22T07:08:00+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/slack.py', '--cfg-path', '/var/folders/1l/hpssfvcn1z7dpsnpgtc10rlw0000gn/T/tmpk7ez3xv2', '--error-file', '/var/folders/1l/hpssfvcn1z7dpsnpgtc10rlw0000gn/T/tmp7n1nuu5n']
[2021-03-22 17:17:29,341] {standard_task_runner.py:77} INFO - Job 28: Subtask send_slack
[2021-03-22 17:17:29,377] {logging_mixin.py:104} INFO - Running <TaskInstance: test_slack.send_slack 2021-03-22T07:08:00+00:00 [running]> on host Choui-MacBook-Pro.local
[2021-03-22 17:17:29,405] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=hana
AIRFLOW_CTX_DAG_ID=test_slack
AIRFLOW_CTX_TASK_ID=send_slack
AIRFLOW_CTX_EXECUTION_DATE=2021-03-22T07:08:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-22T07:08:00+00:00
[2021-03-22 17:17:30,030] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=test_slack, task_id=send_slack, execution_date=20210322T070800, start_date=20210322T081729, end_date=20210322T081730
[2021-03-22 17:17:30,046] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-22 17:17:30,096] {local_task_job.py:146} INFO - Task exited with return code 0
